graylog2-web-interface/src/components/configurations/UrlWhiteListForm.jsx

```diff
@@ -132,7 +134,7 @@ const UrlWhiteListForm = ({ urls, onUpdate, disabled }: Props) => {
           <td>
             <Input type="text"
                    id={`title-input${idx}`}
-                   ref={(elem) => { inputs[`title${idx}`] = elem; }}
+                   ref={(elem) => { inputs[`title${idx}`] = elem; return true; }}
```

Comment:
I'm not sure why we need to return `true` in here, could you please explain it? :)

Reply:
And is it possible to use the `createRef` that I had mentioned in a previous comment?

Reply:
Hi @kyleknighted , these are dynamically created refs for the url whitelist from that's why I use callback ref.